changes/4253-sergeytsaplin.md

```diff
@@ -0,0 +1 @@
+Field aliases are compared with None instead of simple `bool(alias)` because an empty string is a valid JSON attribute name.
```

Comment:
I think we can make this slightly clearer, something like

```suggestion
Allow empty string aliases by using a `alias is not None` check, rather than `bool(alias)`
```